plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -681,10 +681,15 @@ private void addSplitPluginDependencies(String thisPlugin, MavenRunner.Config mc
             }
 
             List<String> convertFromTestDep = new ArrayList<String>();
-            checkDefinedDeps(pluginDeps, toAdd, toReplace, otherPlugins, new ArrayList<String>(pluginDepsTest.keySet()), convertFromTestDep);
+            checkDefinedDeps(pluginDeps, toAdd, toReplace, otherPlugins, new ArrayList<>(pluginDepsTest.keySet()), convertFromTestDep);
```

Comment:
It shouldn't as the method declared the type of the list (it would be equivalent to List<String> vble = new ArrayList<>()).